modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-suma.adoc

```diff
@@ -236,6 +236,22 @@ than 48 characters in length.
 . Open a browser and proceed to your servers FQDN.
 . Enter your username (default is [literal]``admin``) and the password you set during the deployment process.
 
+
```

Comment:
I would reword this section:
[IMPORTANT]
SLE Micro is designed to update itself automatically by default and will reboot after applying updates. However, this behavior is not desirable for the {productname} environment. To prevent automatic updates on your server, we recommend disabling the transactional-update timer.

You can disable the timer by running the following command:

[source, shell]
----
systemctl disable --now transactional-update.timer
----